Improve clarity in the decision making process (#588)
json-schema-org/community: GOVERNANCE.md

## Decision Making

The JSON Schema Project uses 2 different mechanism for decision making:

1. **Consensus**: The JSON Schema Project tries by default to find consensus in all the strategic decision by using the consensus mechanism described below. 

> 2. **Voting**: It is ok use the voting process for those cases when the discussion has already occurred so we can use this simple yet powerful mechanism. 

Reviewer: ```suggestion
2. **Voting**: It is OK use the voting process for those cases when the discussion has already occurred or discussion and discovery is not required. Voting is a simple yet powerful mechanism. 
```
